docker/sample-application/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.8-alpine3.12
```

Comment:
I realize this is a sample app, but we should try to use `registry.access.redhat.com/ubi8/python-38` even though it's not as lean, if we end up using this app more.

Reply:
Good point. I'll have to get used to checking registry.access.redhat.com for images first before I go to dockerhub.

Reply:
Done